bids/analysis/transformations/compute.py

```diff
@@ -35,6 +36,25 @@ def _transform(self, var, model='spm', derivative=False, dispersion=False,
 
         if isinstance(var, SparseRunVariable):
             sr = self.collection.sampling_rate
+            # Resolve diferences in TR and TA to milliseconds
+            trs = {ri.tr for ri in var.run_info}
+            tas = {ri.ta for ri in var.run_info}
+            assert len(trs) == 1
+            assert len(tas) == 1
+            TR = int(np.round(1000. * trs.pop()))
+            TA = int(np.round(1000. * tas.pop()))
+            SR = int(np.round(1000. / sr))
+            if TA is None or TA < TR:
```

Comment:
I don't think `TA` can be `None`, since we run it through `int()`. Do we need to add a check earlier to set `TA` to `TR` if we get `None`?

Reply:
Ah, that's fair. I don't think it actually is possible; I wrote that guard before I wrote the section that sets `ta = tr` in the absence of contrary timing information.